pkg/apiserver/user/sqlauth/sqlauth.go

```diff
@@ -11,6 +15,23 @@ import (
 
 const typeID utils.AuthType = 0
 
+// TiDB config response
+//
+// "security": {
+//   ...
+//   "enable-sem": true/false,
+//   ...
+// },
+type tidbSecurityConfig struct {
+	Security tidbSEMConfig `json:"security"`
+}
+
+type tidbSEMConfig struct {
+	EnableSem bool `json:"enable-sem"`
+}
+
+type set map[string]struct{}
```

Comment:
Maybe this doesn't need a type alias, as it is a common pattern seen everywhere. Adding a new type simply makes people harder to understand what it is.

Reply:
Done.